### PR TITLE
[13.0][MIG] sale_discount_display_amount

### PR DESCRIPTION
**Pull request by Laurent-Corron**

### CODE REVIEW

**Comment by marylla** [comment]
Hi @Laurent-Corron 
I started a functional review with the runbot, but unfortunately I cannot add a new sale order line.
After choosing a product, I immediately get this message:

<pre>
Something went wrong !
sale.order.line(< NewId ref='virtual_224' >,).discount_total
</pre>

File: sale_discount_display_amount/views/sale_view.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<odoo>
+    <record id="sale_order_view_form_inherit_abi_product" model="ir.ui.view">
+        <field name="name">sale.order.form (abi_sale)</field>
```
Comment (legalsylvain) [inline sale_discount_display_amount/views/sale_view.xml]: FYI, this line is useless, because autogenerated.
(not a blocking point)

**Review by legalsylvain — COMMENTED** [review]
Hi,
thanks for this module. LGTM, thanks ! (code review / no test); 

Just a question, i don't see any overload of the sale order report to print quotation.
It should be present in the pdf, don't you think ?
regards.

**Comment by Laurent-Corron** [comment]
@marylla 
Thank you for your test, I will be looking into it soon.

@legalsylvain 
It isn't a need for our client to have the discount displayed in the report, but feel free to make a pull request to add it!

**Comment by legalsylvain** [comment]
> @legalsylvain
It isn't a need for our client to have the discount displayed in the report, but feel free to make a pull request to add it!

No time to do it for the time being. Could you just add a little text in the ROADMAP.rst file, to mention that point.

regards.

**Comment by lmignon** [comment]
@Laurent-Corron Any news about this?

**Comment by lmignon** [comment]
@marylla https://github.com/OCA/sale-workflow/pull/968/commits/85d0584e72e8941cea20c9b1a1e4b73f815bff3d should fix the problem

**Review by marylla — APPROVED** [review]
I tested it again and everything works now. All values were correctly calculated. 
Thank you!

**Comment by lmignon** [comment]
cc @rousseldenis @Cedric-Pigeon

**Review by Cedric-Pigeon — APPROVED** [review]
LGTM, code review

**Review by rousseldenis — APPROVED** [review]

**Comment by Laurent-Corron** [comment]
ping @rousseldenis

**Pull request merged**